vnet/lib/vnet/endpoints/1.0/topologies.rb

```diff
@@ -28,6 +28,31 @@ def self.put_post_shared_params
     update_by_uuid(:Topology)
   end
 
+  param_uuid M::Interface, :interface_uuid, required: true
+  post '/:uuid/datapaths/:datapath_uuid' do
+    topology = uuid_to_id(M::Topology, "uuid", "topology_id")
+    datapath = uuid_to_id(M::Datapath, "datapath_uuid", "datapath_id")
+    interface = uuid_to_id(M::Interface, "interface_uuid", "interface_id")
+
+    remove_system_parameters
+
+    result = M::TopologyDatapath.create(params)
+    respond_with(R::TopologyDatapath.generate(result))
+  end
+
+  get '/:uuid/datapaths' do
+    show_relations(:Topology, :topology_datapaths)
+  end
+
+  delete '/:uuid/datapaths/:datapath_uuid' do
+    topology = check_syntax_and_pop_uuid(M::Topology)
+    datapath = check_syntax_and_pop_uuid(M::Datapath, 'datapath_uuid')
+
+    M::TopologyDatapath.destroy(topology_id: topology.id, datapath_id: datapath.id)
+
+    respond_with([datapath.uuid])
+  end
+
```

Comment:
Unit tests need to be added for these new routes. Something similar to [this](https://github.com/axsh/openvnet/blob/develop/vnet/spec/vnet/endpoints/1.0/datapaths_spec.rb#L47-L57) should be enough.